packages/ui/src/components/Text/index.tsx

```diff
@@ -121,6 +134,7 @@ export const Text = ({
   as,
   color = 'neutral',
   oneLine = false,
+  placement = 'start',
```

Comment:
mb we could keep it undefined if no provided and let natural browser behavior do its job ? + there would be less "noise" in snapshots